ooniprobe/PrefixHeader.pch

```diff
@@ -10,6 +10,8 @@
 // Include any system framework and library headers here that should be included in all compilation units.
 // You will also need to set the Prefix Header build setting of one or more of your targets to reference this file.
 
+#define COLLECTOR_ADDRESS @"https://b.collector.ooni.io"
+
```

Comment:
I wonder what change in Xcode was such that this ended up here 🤔

Reply:
See above. There is an option in the settings to set the default collector in case you mess with the settings

Reply:
Okay